recipes/read-excel/all/conanfile.py

```diff
@@ -27,27 +27,6 @@ def _compilers_minimum_version(self):
             "apple-clang": "10"
         }
 
-    @property
-    def _module_subfolder(self):
-        return os.path.join("lib", "cmake")
-
-    @property
-    def _module_file_rel_path(self):
-        return os.path.join(self._module_subfolder,
-                            "conan-official-{}-targets.cmake".format(self.name))
-
-    @staticmethod
-    def _create_cmake_module_alias_targets(module_file, targets):
-        content = ""
-        for alias, aliased in targets.items():
-            content += textwrap.dedent("""\
-                if(TARGET {aliased} AND NOT TARGET {alias})
-                    add_library({alias} INTERFACE IMPORTED)
-                    set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
-                endif()
-            """.format(alias=alias, aliased=aliased))
-        tools.save(module_file, content)
-
```

Comment:
why did you remove all code related to the cmake alias targets ?

Reply:
Because it's not very good to use a library in CMake without namespace. I did so in all my libraries. Minor change for the users, but more standardized.